res/TensorFlowLiteRecipes/BroadcastTo_000/test.recipe

```diff
@@ -15,11 +15,11 @@ operand {
   shape { dim: 1 dim: 2 dim: 3 }
 }
 operation {
-  type: "Custom"
+  type: "BroadcastTo"
   input: "bc_input"
   input: "bc_shape"
   output: "bc_ofm"
-  custom_code: "BroadcastTo"
+  version: 2
```

Comment:
Q) why is this changed?

Reply:
@seanshpark 
> Q) why is this changed?

`BroadcastTo` was newly introduced as a built-in op, so I modified it to fit the existing broadcastTo_000 recipe file. 
Would it be better to make `BroadcastTo_001` and `BroadcastTo_002` recipe files separately for Builtin op test use?

Reply:
> Would it be better to make BroadcastTo_001 and BroadcastTo_002 recipe files separately for Builtin op test use?

Yes

Reply:
@seanshpark 

Thank you for your advice !! 
I applied comments. :)